src/sass/_init.scss

```diff
@@ -2,3 +2,6 @@
 @import 'abstractions/colors';
 @import 'abstractions/variables';
 @import '~susy';
+
+// Uncomment next @import if you need trimming feature on the project.
+//@import 'line-clamp';
```

Comment:
if we don't need this feature for all projects, maybe have sense to make it as component and move outside the theme? somewhere to Skilld component library, and load it only when we need it

Reply:
why? it's disabled anyway

Reply:
why we need code that we don't need/use? it's more logical to import it like component or library.